src/components/SegmentedProgressBar/SegmentedProgressBar.jsx

```diff
@@ -17,7 +17,7 @@ export default function SegmentedProgressBar({ current, total, label }) {
       aria-valuemin="0"
       aria-valuemax={total}
       tabIndex="0"
-      aria-label={label}
+      aria-label={`Step ${current} of ${total}`}
```

Comment:
I looked more closely this morning for instances of `<SegmentedProgressBar />` to make sure we wouldn't introduce trouble upstream. I found only three, and no instances were passing in a `{label}` prop. The most pertinent one was [line 83 of FormNav.jsx](https://github.com/department-of-veterans-affairs/vets-website/blob/9c610f8981da2b3608af0e4bc10519072d78d158/src/platform/forms-system/src/js/components/FormNav.jsx#L83). I think this is a good solution, and definitely could be iterated on down the road.